### PR TITLE
Fixing some typos in index.html and creating README file

### PR DESCRIPTION
**Pull request by maker2413**
Hi :wave:, I just noticed some minor typo mistakes in the website and thought I would try to fix them as I love the project. I also created a README so the project is described on GitHub when people review the repo.

If you want me to drop the README or aren't interested in my PR that is fine I just really like the idea of the project so I thought I would try to help out.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Emacs Tales was inspired by amazing, exciting stories I frequently read on
+[[https://www.reddit.com/r/emacs][r/emacs]]. One special [[https://www.reddit.com/r/emacs/comments/k2oppg/what_hooked_you_on_emacs/gdwhm5r][comment]] triggered the idea for a place to collect stories
+like that, so they don't get lost on Reddit.
+
+I believe that share this personal tales makes the emacs experience more human
+and helps we develop ourselves and our emacs skills together.
+
+Feel free to share you tale with us [[https://github.com/emacs-love/tales][opening a pull request]] or sending your story
```
Comment (jiacai2050) [inline README.org]: with us by ....

File: README.org [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Emacs Tales was inspired by amazing, exciting stories I frequently read on
+[[https://www.reddit.com/r/emacs][r/emacs]]. One special [[https://www.reddit.com/r/emacs/comments/k2oppg/what_hooked_you_on_emacs/gdwhm5r][comment]] triggered the idea for a place to collect stories
+like that, so they don't get lost on Reddit.
+
+I believe that share this personal tales makes the emacs experience more human
```
Comment (jiacai2050) [inline README.org]: this -> those

Comment (maker2413) [reply README.org]: Thanks for catching this!

**Comment by guilhermecomum** [comment]
Sorry for the delay, @maker2413 and @jiacai2050. Thank you so much for the contribution. :D

**Pull request merged**